projects/rocprofiler-systems/source/lib/rocprof-sys/library/rocprofiler-sdk.cpp

```diff
@@ -1128,6 +1408,69 @@ tool_tracing_callback(rocprofiler_callback_tracing_record_t record,
                 }
             }
             break;
+#if(ROCPROFILER_VERSION >= 600)
+            case ROCPROFILER_CALLBACK_TRACING_OMPT:
+            {
+                // OMPT callbacks that when received, should call start followed
+                // immediately by stop This mimics the timemory implementation
+                static std::set<rocprofiler_ompt_operation_t> ompt_instant_events = {
+                    ROCPROFILER_OMPT_ID_dispatch,
+                    ROCPROFILER_OMPT_ID_mutex_acquire,
+                    ROCPROFILER_OMPT_ID_flush,
+                    ROCPROFILER_OMPT_ID_cancel,
+                    ROCPROFILER_OMPT_ID_device_initialize,
+                    ROCPROFILER_OMPT_ID_device_finalize,
+                    ROCPROFILER_OMPT_ID_device_load,
+                    // ROCPROFILER_OMPT_ID_device_unload // Unsupported by runtime
+                };
+
+                // Callbacks that are received but that we do not process
+                static std::set<rocprofiler_ompt_operation_t> ompt_no_process = {
```

Comment:
```suggestion
                static const std::set<rocprofiler_ompt_operation_t> ompt_no_process = {
```
It looks like this can be a `const`.